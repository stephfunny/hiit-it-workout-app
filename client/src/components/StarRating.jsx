// import React from 'react';
// import ReactDOM from 'react-dom';
// import ReactStarRating from 'react-star-rating-component';
//
// var StarRating = (props) => (
//
//   <ReactStarRating
//     editing={false}
//     emptyStarColor="#E3E3E3"
//     name="rating"
//     starColor="#238A88"
//     starCount={5}
//     value={props.rating}
//   />
//
// );
//
// export default StarRating;
