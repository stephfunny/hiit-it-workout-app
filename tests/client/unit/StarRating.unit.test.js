import React from 'react';
import { configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import StarRating from '../../../client/src/components/StarRating.jsx';
import bookingsSample from '../../../database/dummy_data.js';

configure({ adapter: new Adapter() });

describe('StarRating', () => {
  it('should render correctly', () => {
    const output = shallow(
      <StarRating rating={bookingsSample.star_rating}></StarRating>
    );
    expect(shallowToJson(output)).toMatchSnapshot();
  });
});