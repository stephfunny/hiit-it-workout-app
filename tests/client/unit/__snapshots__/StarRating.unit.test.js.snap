// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StarRating should render correctly 1`] = `
<div>
  <StarRatingComponent
    editing={false}
    emptyStarColor="#333"
    name="rating"
    starColor="#ffb400"
    starCount={5}
    value={0}
  />
  <span>
    # Ratings
  </span>
</div>
`;
